secant/Stubs/MockServices.swift

```diff
@@ -58,18 +58,24 @@ class MockMnemonicPhraseHandling: MnemonicSeedPhraseHandling {
 }
 
 class KeysPresentStub: KeyStoring {
+    var returnBlock: () throws -> Bool
+    var called = false
```

Comment:
Now that you've moved this, maybe this should be called areKeysPresentCalled or something